models/_sector/dex/trades/bnb/dex_bnb_base_trades.sql

```diff
@@ -0,0 +1,47 @@
+{{ config(
+    schema = 'dex_bnb',
+    alias = 'base_trades',
+    materialized = 'view'
+    )
+}}
+
+
+-- (blockchain, project, project_version, model)
+{% set base_models = [
+    ref('uniswap_v3_bnb_base_trades')
+] %}
+
+WITH base_union AS (
+    SELECT * FROM (
+        {% for base_model in base_models %}
+        SELECT
+            blockchain,
```

Comment:
why not just `select *` as schema already defined and determined in macro?

Reply:
We try to list all columns in all roll-up spells, to avoid column mismatching. There were bugs happened before and they were very hard to identify.

Reply:
as far as they use one macro and there aren't if-else conditions on columns, it should be fine. but yes, i agree that explicit is better then implicit :)